examples/spark/pom.xml

```diff
@@ -38,6 +38,12 @@
       <groupId>org.apache.carbondata</groupId>
       <artifactId>carbondata-spark_${spark.binary.version}</artifactId>
       <version>${project.version}</version>
+      <exclusions>
```

Comment:
Was wondering why this exclusion in examples/spark/pom.xml & integration/spark/pom.xml . You don't seem to have any change in these 2 modules. I think, you want to exclude elsewhere ?

Reply:
Without exclusion, all these dependencies was causing transitive dependency and because of that build was failing.